package/environments/base.js

```diff
@@ -99,82 +84,36 @@ const getBaseConfig = () => new ConfigObject({
     path: config.outputPath,
     publicPath: config.publicPath
   },
-
+  entry: getEntryObject(),
   resolve: {
     extensions: config.extensions,
-    plugins: [PnpWebpackPlugin]
+    modules: getModulePaths()
   },
 
+  plugins: getPlugins(),
+
+  optimization: {
+    splitChunks: {
+      chunks: 'all',
+      name: true
+    },
+    runtimeChunk: 'single'
+  },
```

Comment:
Thinking of leaving `òptimization.splitChunks` on by default? That would be interesting! Perhaps we might consider then swapping out the implementation of `javascript_pack_tag` with that of `javascript_packs_with_chunks_tag` to make that the default behavior.

Reply:
Yeah, I think that makes sense for proper long term caching support (this is recommended in webpack docs)